dash/linux-dash.yml

```diff
@@ -0,0 +1,11 @@
+dash:
+  image: husseingalal/busydash
+  privileged: true
+  links:
+  - network
+  labels:
+  - io.rancher.os.scope=system
+  restart: always
+  pid: host
+  ipc: host
+  net: host
```

Comment:
@ibuildthecloud, will this open the port on the host/system-docker? So instead of specifying a port mapping (which is currently broken), we can just attach the container to the hosts network?